### DISCUSSION TITLE
Nitro without Nuxt3

### DISCUSSION THREAD

**Discussion by SimoMay** [nitrojs/nitro#233] [Q&A] [answered]
Hey,

Is this project meant to be used separately from Nuxt3?

I'm trying to deploy it to Vercel and it does not recognize it, 
So I sat manually:
- NITRO_PRESET to vercel 
- output folder to be `.vercel_build_output`
- build command to be `npx nitropack build`

And still not recognized, the error I get are mostly 
```
ERROR	ReferenceError: defineEventHandler is not defined    at Object.<anonymous> (/vercel/path2/api/config.ts:2:1)
```

Also, separate question, I see you using `c12` and `dotenv` (installed but never used in code as far as I can see) but I can't get the .env file to load in config, the only way that work is to add an `extends` with a ts file

**Comment by danielroe** [comment, marked as answer]
You can see https://github.com/unjs/nitro-deploys for an example project that's being deployed to Vercel...

**Reply by SimoMay** [reply]
I tried to match [unjs/nitro-deploys](https://github.com/unjs/nitro-deploys) as much as I could find (`vercel.json` scripts in `package.json`, even `tsconfig.json`) the thing that help was moving my `api/` folder into my `routes/` folder.

Given this issue, I believe the `api` folder shortcut should be removed from the documentation if it causes problems.

Thanks for pointing me to the example repo @danielroe

**Reply by pi0** [reply]
Indeed thats a pain. We added a small note in the docs regarding this:

> Note: api/ is a shortcut for routes/api as a common prefix. However, please note that some deployment providers use app/ directory for their API format. You can simply use the routes/api or srcDir option to move everything under src/ or server/ directory.

https://nitro.unjs.io/guide/routing.html

PR more than welcome for improving notice.

**Reply by SimoMay** [reply]
I think the place for the notice is in every provider doc that's affected by this, which would defeat the wording of "zero config deployment" that's why I think the solution is not in the doc, it's removing the shortcut for api folder.

But notice on the providers pages could be a mid solution too.

**Comment by rieger-jared** [comment]
Hi After running into the same issue, I've updated the docs via this PR https://github.com/unjs/nitro/pull/1386
